easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
+
+name = 'pytest'
+version = '3.8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pytest.org'
+description = """pytest: simple powerful testing with Python"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [('Python', '3.6.4')]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_list = [
+    ('py', '1.6.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/py/'],
+        'checksums': ['06a30435d058473046be836d3fc4f27167fd84c45b99704f2fb5509ef61f9af1'],
+    }),
+    ('attrs', '18.2.0', {
+        'modulename': 'attr',
+        'source_urls': ['https://pypi.python.org/packages/source/a/attrs/'],
+        'checksums': ['10cbf6e27dbce8c30807caf056c8eb50917e0eaafe86347671b57254006c3e69'],
+    }),
+    ('pluggy', '0.7.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pluggy/'],
+        'checksums': ['95eb8364a4708392bae89035f45341871286a333f749c3141c20573d2b3876e1'],
+    }),
+    ('atomicwrites', '1.2.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/atomicwrites/'],
+        'checksums': ['ec9ae8adaae229e4f8446952d204a3e4b5fdd2d099f9be3aaf556120135fb3ee'],
+    }),
+    ('more-itertools', '4.3.0', {
+        'modulename': 'more_itertools',
+        'source_urls': ['https://pypi.python.org/packages/source/m/more-itertools/'],
+        'checksums': ['c476b5d3a34e12d40130bc2f935028b5f636df8f372dc2c1c01dc19681b2039e'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pytest/'],
+        'checksums': ['a8a07f84e680482eb51e244370aaf2caa6301ef265f37c2bdefb3dd3b663f99d'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/pytest'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
this line can also be removed with PythonBundle.

Reply:
@smoors I finally got around to addressing all your comments in this PR.  Please let me know if you have any other comments.